cosmo_tester/framework/util.py

```diff
@@ -302,12 +302,48 @@ def get_plugin_wagon_urls():
     return yaml.load(requests.get(plugin_urls_location).text)['plugins']
 
 
-def get_cli_package_urls():
-    if is_community():
-        filename = 'cli-packages.yaml'
+def test_cli_package_url(url):
+    error_base = (
+        # Trailing space for better readability when cause of error
+        # is appended if there are problems.
+        '{url} does not appear to be a valid package URL. '
+    ).format(url=url)
+    try:
+        verification = requests.head(url)
```

Comment:
maybe wanna `allow_redirects=True`? i'm not sure we want to also RuntimeError on a 304, instead of just following the redirect

Reply:
It defaults to allow_redirects=True- I double checked that before I put it in. Is it worth specifying given that?

Reply:
it doesn't according to the docs http://docs.python-requests.org/en/latest/user/quickstart/#redirection-and-history

Reply:
That says it does default to True but that you can override it.

Also, from help(requests.request):
    :param allow_redirects: (optional) Boolean. Enable/disable GET/OPTIONS/POST/PUT/PATCH/DELETE/HEAD redirection. Defaults to ``True``.

And from running in an interpreter in the env with the tests:
>>> requests.get('https://www.google.com').status_code
200
>>> requests.get('https://www.google.com', allow_redirects=False).status_code
302

Reply:
Please note that you used `.get` to check, and `.head` in the actual code. From the docs:
>By default Requests will perform location redirection for all verbs except HEAD.

Reply:
Yeah, Lukasz is right:
```
requests.head('https://www.google.com').status_code
302

requests.head('https://www.google.com', allow_redirects=True).status_code
200
```